pennylane/ops/qubit/parametric_ops.py

```diff
@@ -55,6 +55,9 @@ class RX(Operation):
     grad_method = "A"
     generator = [PauliX, -1 / 2]
 
+    def __init__(self, phi, wires):
+        super().__init__(phi, wires=wires, do_queue=True, id=None)
```

Comment:
@Jaybsoni flyby comment, but this allows us to now delete the signature 'override' on line 34 🙌